getting-started/templates/systemlink-values.yaml

```diff
@@ -587,25 +568,34 @@ dashboardhost:
         ## <ATTENTION> - Uncomment to set the list of unsigned plugins to load. This will override the plugins included by default so ensure you maintain
         ## the list of plugins listed in the default values file.
         ##
-        # allow_loading_unsigned_plugins: ni-slnotebook-datasource,ni-sldataframe-datasource,ni-plotly-panel
+        # allow_loading_unsigned_plugins: ni-systemlink-app
 
 ## Grafana provisioning
 ##
 # grafanaprovisioning:
   # grafanaDashboardsProvisioning:
     ## Dashboards listed are enumerated over the dashboard-configmap-template.yaml
-    ## Each key under the "dashboards" item will be the name of a ConfigMap.
+    ## Each key in the under the "dashboards" item will be the name of a ConfigMap.
```

Comment:
Here are some recommendations:

> Each key under the dashboards item uses the name of a ConfigMap.
> NI recommends adding the "-dashboard" suffix to each key to avoid a file collision with other ConfigMaps.
> You can place all Dashboard JSON files under the "dashboards/[project-name]/" file path.
> By default, SystemLink considers all enabled parameters as ‘True,’ even if a parameter is not in the JSON file. You can disable this setting by assigning the ‘whitelistMode’ value to ‘True.’